src/main/groovy/au/org/ala/spatial/process/SlaveProcess.groovy

```diff
@@ -932,6 +936,7 @@ class SlaveProcess {
             String fq = ''
             if (extraFq) fq = '&fq=' + URLEncoder.encode(extraFq, "UTF-8")
             String url = species.bs + "/occurrences/facets/download?facets=names_and_lsid&lookup=" + lookup + "&count=" + count + "&q=" + species.q + fq
+            log.info("Loading species from: " + url )
```

Comment:
Also append to the task log, maybe without the URL.

Reply:
Done